### ISSUE TITLE
Prevent or Supress Key Events

### ISSUE THREAD

**Issue by libmysterion** [octalmage/robotjs#220]
Hi I'm building a remote control app (keyboard/mouse sharing)which uses electron and robotjs. I've done this before and one issue we hit early on was that we needed to suppress key events on the host machine i.e. the one which is transmitting its input to the client app; in particular the windows key. I [found a way](https://github.com/libmysterion/kvm/blob/master/kvm-common/src/main/java/com/mystery/kvm/common/transparentwindow/KeyHook.java) to do it using JNI but I'd like to be able to do this using pure node and the sensible place for it to live seems to be robotjs. 

**TLDR**; This is an enhancement request to add the ability to block and unblock specified keys, the primary use case is to block 'command' in Remote Desktop type apps.

**Comment by libmysterion** [comment]
I would be interested in performing this work if it might be accepted in a PR.

I am thinking what I would like in the API would be something like:

robotjs.setKeyBlocked('command', true); // to block
robotjs.setKeyBlocked('command', false); // to unblock

Would you accept such a PR if it were up to scratch?

**Comment by octalmage** [comment]
I would totally accept it as long as it was cross platform! That's the main rule I'm trying to keep.

**Comment by octalmage** [comment]
Generally if you use an electron window to take the input, the hotkeys are captured. How are you using this?

**Comment by libmysterion** [comment]
I am able to capture the "windows" key in an electron app with a jquery handler on the window in the renderer but cant find any way suppressing the system event

**Comment by oktapodia** [comment]
This issue has been closed because it has been inactive for > 180 days.
    You may reopen this issue if it has been closed in error.

**Comment by L1lith** [comment]
I'm still having this issue. I'm trying to write a script to re-bind the keys on the keyboard to change the layout in realtime. This feature doesn't seem to exist on NPM at all and would be sooo useful!
